packages/netsuite-adapter/src/client/client.ts

```diff
@@ -203,6 +205,56 @@ export default class NetsuiteClient {
       .toArray()
   }
 
+  private static async createDependencyMap(
+    changes: ReadonlyArray<Change>,
+    deployReferencedElements: boolean,
+    elementsSourceIndex: LazyElementsSourceIndexes,
+  ):Promise<collections.map.DefaultMap<string, Set<string>>> {
+    const dependencyMap = new collections.map.DefaultMap<string, Set<string>>(() => new Set())
+    const elemIdsAndCustInfoArr = await awu(changes)
+      .map(getChangeData)
+      .map(async instance => ({
+        elemId: instance.elemID,
+        custInfo: await NetsuiteClient.toCustomizationInfos(
```

Comment:
its an array of customization infos
```suggestion
        custInfos: await NetsuiteClient.toCustomizationInfos(
```